cmd/initialize/bootstrap.go

```diff
@@ -14,19 +14,25 @@ import (
 
 const EnvFileName string = ".env"
 
-func bootstrapFiles(m *manifest.Manifest, toolID string, basePath string) error {
+func BootstrapFiles(m *manifest.Manifest, toolID string, basePath string) error {
```

Comment:
Should this maybe be moved to `internal`?

Reply:
Yes it could, but lets not do it here, since I did not change this.